### DISCUSSION TITLE
Should we and how could we celebrate Escape Velocity’s 30th birthday?

### DISCUSSION THREAD

**Discussion by brendanjones** [endless-sky/endless-sky#10088] [Ideas]
I’m a bit ahead of time here, but the original Escape Velocity will celebrate its 30th birthday in 2026. May 4 2026, if [this page](https://kotaku.com/games/escape-velocity) is correct. 

As someone who’s dropped thousands of hours into all the EV titles since the age of 10, I think it’d be really fun to celebrate this milestone of the game that inspired our beloved Endless Sky. Do you guys agree?

If so, what could we do to honour EV? I’m thinking along the lines of adding some temporary content to the game, like for a month or so (which could afterwards be made available in a plugin?). Wouldn’t it be fun to fly around in a Confed Cruiser?!

Other ideas (also temporary additions):
- Could be nice to add an EV-birthday-celebrating skin to the loadout screen. Bonus points for playing the [EV theme song](https://m.youtube.com/watch?v=lkq06k1JA4E) (dunno if there are copyright issues but surely we could get permission). 
- Add a mission that in some way references EV
- Add a planet with ships and outfits from EV
- …? Go wild with your suggestions! We’ve got plenty of time to plan this :)

**Comment by brendanjones** [comment]
I also understand if people say ES isn’t EV and let’s keep EV out of ES. I’m sure there are many players who weren’t even alive when EV came out, let alone played it! Celebrating EV’s bday should only be done if people think it’s a fun (and somewhat nostalgic) thing.

**Comment by Saugia** [comment]
We don't really do temporary content. This has been mentioned before with things like April Fools or other holidays. If we add anything to vanilla, generally, it's meant to be a permanent part of it (as in, we wouldn't put something in with the idea to remove it completely after some time, but of course subject to adjustments, etc.).

For your suggestions: 

Skins like the above we don't do.
The EV theme song would run into copyright, yes.
A small mission that references it would be most plausible; not one that uses an existing character or ship, but something that might reference a small mission in EV.
We wouldn't add outfits specifically from any of the EV games. Technically some of our outfits and ships are similar or a few are also references, but we should add outfits with reason, not explicitly for a reference.

We also have EV's discord linked in our own discord, I'm sure they'd be more than happy to hear about things like this. That said, for EV, the original game, it's pretty much impossible to contact Matt Burch these days. Evopac (EV Override, working on Cosmic Frontier), and David (lead of ATMOS for EV Nova) you can talk to in the EV discord; I've chatted with them a few times myself.

I've talked to them about having plug-ins for ES that emulate EV games; Evopac seemed okay with it if it was transformative enough (which I've created a plug-in for some EVO content), but Dave preferred this isn't done and you make your own content. So, that said, it would seem inappropriate to use anything directly EV in ES, but references could be considered.

**Comment by Zitchas** [comment]
Personally, I'd be interested in a follow up mission to the Kestrel series. It's already an EV reference, having a mission that references something in EV or AmbrosiaSW would be quite interesting, and could tie into the date.

Maybe on May 4th every year there's a mission available for people flying Kestrels where a parrot named Hector asks the player to take him for a spin through nearby systems, maybe blow up some pirates. Just a random idea. I'm interested.

**Reply by brendanjones** [reply]
I like this, it doesn’t add anything that breaks ES immersion but still honours EV. 

If one of those pirates was called Harry The Handsome Executive I’d be overjoyed, that game was the best.

**Reply by Zitchas** [reply]
Oh, definitely. There's so many bits of lore one could reference in there. Shareware distro CDs, for instance, such as MacFormat or MacAddict.

Maybe the parrot's owner is a collector of antique computer stuff? Sends the player on a hunt for vintage systems, including these giant boxy computers  in gaudy colors...

**Reply by SpearDane** [reply]
I know it's the wrong EV, but a dead world with strange wooden ships and broken shackles on it would be fun. ;)

**Reply by Zitchas** [reply]
Which EV is that?

**Reply by brendanjones** [reply]
I was thinking @SpearDane is referring to the race in Override that fights the Voinians in wooden ships  https://evn.fandom.com/wiki/Emalgha_Fighter . Correct, Dane?

**Reply by Zitchas** [reply]
Oh, the Emalgha!  Yes, from EVO. Hmmm... Probably wouldn't want them in an EV specific anniversary thing, but in a larger "tribute to the EV series in general" easter egg? Sure, that'd be great. Maybe see a poster of some highly unlikely wooden spaceships in a place the pilot passes through on a mission...

**Reply by SpearDane** [reply]
Yeaah the Emalgha! Sorry I never got into EV. I played EVO first and when checked out EV I was turned off by how small it felt (one of my most cherished moments in EVO was spending so much time working on the earth-voinian stuff, and southern traders, and feeling like I'd completed a LOT, just to discover the the Minaruu and realize there really was an entire universe out there I never touched. This was also something I missed in EVN).

You could wander into a crazy person talking to themselves, mumbling about the EV rebels or something.

**Reply by brendanjones** [reply]
All good, you are forgiven ;) I can imagine not getting into EV if EVO was your first experience. I played EV when it came out and I was 10, so I have a massive nostalgic soft spot for it given it blew my mind at the time, but EVO is a far superior game. 

I mean how damn cool were all the Crescent ships?! I can’t wait for that EVO reboot - I donated to the kickstarter but it looks like development has slowed. All good, when it comes it comes.

**Reply by Zitchas** [reply]
Development might have slowed (and definitely missed the original goal), but what's been shown so far (especially the artwork) has been phenomenal, and still making progress. Just slower.

In any case, lots of good ideas for ways that could subtly reference / pay tribute to classic EV (as well as sequels on their own dates), people just need to do them.
